warehouse/ingest-scripts/src/main/resources/bin/ingest/create-tomorrows-shards.sh

```diff
@@ -22,23 +34,37 @@ if [[ "$2" == "" ]]; then
 else
   declare -i divisor=$2
 fi
-numshards=$((numshards * numerator / divisor))
 
-shardsPerSplit=1
-if [[ -n "$3" ]]; then
-    shardsPerSplit=$3
+if [[ "$3" == "" ]]; then
+  declare -i errorTableShards=$numshardsOrg
+else
+  declare -i errorTableShards=$3
 fi
 
+if [[ "$4" == "" ]]; then
+  declare -i queryMetricShards=$numshardsOrg
+else
+  declare -i queryMetricShards=$4
+fi
+
+shardTableNumShards=$((numshardsOrg * numerator / divisor))
+errorTableNumShards=$((errorTableShards * numerator / divisor))
+queryMetricTableNumShards=$((queryMetricShards * numerator / divisor))
+
 DATE=`date -d tomorrow +%Y%m%d`
 
-echo "Generating ${numerator}/${divisor} of ${NUM_SHARDS} = $numshards shards for $DATE"
+echo "Generating ${numerator}/${divisor} of ${numshardsOrg} = $shardTableNumShards shards for $DATE"
+echo "Generating ${numerator}/${divisor} of ${errorTableShards} = $errorTableNumShards shards for $DATE"
+echo "Generating ${numerator}/${divisor} of ${queryMetricShards} = $queryMetricTableNumShards shards for $DATE"
 
 TYPES=${BULK_INGEST_DATA_TYPES},${LIVE_INGEST_DATA_TYPES},${COMPOSITE_DATA_TYPES}
 
 ADDJARS=$THIS_DIR/$DATAWAVE_INGEST_CORE_JAR:$THIS_DIR/$COMMON_UTIL_JAR:$THIS_DIR/$DATAWAVE_CORE_JAR:$THIS_DIR/$DATAWAVE_COMMON_UTILS_JAR:$THIS_DIR/$COMMONS_LANG_JAR
 
-CLASSPATH=$ADDJARS $WAREHOUSE_ACCUMULO_HOME/bin/accumulo datawave.ingest.util.GenerateShardSplits $DATE 1 ${numshards} ${shardsPerSplit} -addShardMarkers -addDataTypeMarkers $TYPES $USERNAME $PASSWORD ${SHARD_TABLE_NAME} $WAREHOUSE_INSTANCE_NAME $WAREHOUSE_ZOOKEEPERS
+daysToGenerate=1
+
+CLASSPATH=$ADDJARS $WAREHOUSE_ACCUMULO_HOME/bin/accumulo datawave.ingest.util.GenerateShardSplits $DATE ${daysToGenerate} ${shardTableNumShards} ${SHARD_TABLE_SHARDNUM_SPLIT_STEP} -addShardMarkers -addDataTypeMarkers $TYPES $USERNAME $PASSWORD ${SHARD_TABLE_NAME} $WAREHOUSE_INSTANCE_NAME $WAREHOUSE_ZOOKEEPERS
 
-CLASSPATH=$ADDJARS $WAREHOUSE_ACCUMULO_HOME/bin/accumulo datawave.ingest.util.GenerateShardSplits $DATE 1 ${numshards} ${shardsPerSplit} -addShardMarkers -addDataTypeMarkers $TYPES $USERNAME $PASSWORD ${ERROR_SHARD_TABLE_NAME} $WAREHOUSE_INSTANCE_NAME $WAREHOUSE_ZOOKEEPERS
+CLASSPATH=$ADDJARS $WAREHOUSE_ACCUMULO_HOME/bin/accumulo datawave.ingest.util.GenerateShardSplits $DATE ${daysToGenerate} ${errorTableNumShards} ${ERROR_SHARD_TABLE_SHARDNUM_SPLIT_STEP} -addShardMarkers -addDataTypeMarkers $TYPES $USERNAME $PASSWORD ${ERROR_SHARD_TABLE_NAME} $WAREHOUSE_INSTANCE_NAME $WAREHOUSE_ZOOKEEPERS
```

Comment:
Did you update the ingest_env.sh with the new properties?

Reply:
In the latest commit, slipped through the cracks there. thanks